### PR TITLE
Added file on movedim

### LINKED ISSUE #5631: [Term Entry] PyTorch Tensor Operations .movedim()
### Reason/inspiration (optional)

We would like a new entry on the `.movedim()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Neema-Joju**
### Description

Added a new file entry on movedim(). Improving documentation and providing more context for the users to understand.

### Issue Solved

Closes #5631 
### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+## Syntax
+
+```pseudo
+torch.movedim(input, source, destination)
+```
+
+- `input`: The input tensor.
+- `source`: Original position of the dimensions that need to move. It must be unique.
+- `destination`: Destination position for each of the original dimension. It must be unique.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md:27]:
```suggestion
- `input`: The input tensor whose dimensions are to be rearranged.
- `source`: The dimensions to be moved. Can be a single integer or a tuple of integers.
- `destination`: The target positions for the dimensions specified in `source`. It should have the same length as `source`.
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md:27]: reframed for better clarity

File: content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+print(c1)
+print("\n")
+
+# Move dimension 0 to dimension 2 for 3D tensor
+c2 = torch.movedim(c, 0, 2)
+print("Three Dimensional tensor (Dim 2):")
+print(c2)
+print("\n")
+
+# Move dimensions [0, 1] to positions [2, 3]
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md:75]:
```suggestion
# Move dimensions [0, 1] to positions [2, 3] for 4D tensor
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Neema-Joju thank you for contributing to Codecademy Docs, the entry is nicely written! 😄 

I've suggested a few changes, could you please review and modify those at your earliest convenience? Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the changes @Neema-Joju 😄 

The entry looks good for a second round of review! 🚀

File: content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+print("Three Dimensional tensor (Dim 2):")
+print(c2)
+print("\n")
+
+# Move dimensions [0, 1] to positions [2, 3] for 4D tensor
+d1 = torch.movedim(d, [0, 1], [2, 3])
+print("Four Dimensional tensor:")
+print(d1)
+```
+
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md:80]:
```suggestion
This example will generate the following output:

```

File: content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+print("Three Dimensional tensor (Dim 2):")
+print(c2)
+print("\n")
+
+# Move dimensions [0, 1] to positions [2, 3] for 4D tensor
+d1 = torch.movedim(d, [0, 1], [2, 3])
+print("Four Dimensional tensor:")
+print(d1)
+```
+This example will generate the following output:
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/movedim/movedim.md:80]:
```suggestion

This example will generate the following output:
```

**Review by Radhika-okhade — APPROVED** [review]

**Pull request merged**